### PR TITLE
Move things around

### PR DESCRIPTION
**Pull request by samelhusseini**
- [x] Move things around so all packages are under a single ``plugins`` directory.
- [x] Update README with correct folder shortcut
- [ ] Update root README with all packages
- [ ] Update all plugin package.json with folder structure
- [ ] Update templates

### CODE REVIEW

**Comment by samelhusseini** [comment]
@alschmiedt I'm going to complete the changes to package.json and readmes in a different PR so that its easier to parse. Review this PR only for folder structure + naming of folders.

File: plugins/README.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ tag on [npm](https://www.npmjs.com/search?q=%40blockly).
 
 ### Plugins
 
-- [``@blockly/plugin-workspace-search``](plugins/workspace-search/): A plugin that adds workspace search support.
+- [``@blockly/plugin-workspace-search``](blockly-workspace-search/): A plugin that adds workspace search support.
```
Comment (alschmiedt) [inline plugins/README.md]: Do we want to addd typed modal and the mutator package to this list?

**Review by alschmiedt — APPROVED** [review]
Curious if we need blockly-  in front of all of our plugin packages names? (ex: blockly-dev-scripts)

File: plugins/README.md (hunk 1 above)
Comment (samelhusseini) [reply plugins/README.md]: Yes, I'll add to my TODO list of readme updates.

**Comment by samelhusseini** [comment]
Removed blockly- prefix

**Review by alschmiedt — APPROVED** [review]

**Pull request merged**